src/core/Track.cpp

```diff
@@ -572,7 +580,7 @@ void TrackContentObjectView::leaveEvent( QEvent * e )
 {
 	if( cursor().shape() != Qt::BitmapCursor )
 	{
-		setCursor( QCursor( embed::getIconPixmap( "hand" ), 3, 3 ) );
+		setCursor( QCursor( embed::getIconPixmap( "hand" ), m_mouseHotspotX, m_mouseHotspotY ) );
```

Comment:
Looking at this now, I think we need to add `3` and `3` in the "classic" theme CSS?

Reply:
Correct, and I believe the new cursor was using something like 3, 7 last I tried it.

Reply:
How do you find out the right coordinates? Opening both images in gimp/etc and just use where the finger points? If yes, I think it's (2,7) for the default, and (3,3) for the classic.

Reply:
Right, 2,7. I've added the settings for classic too and this works fine.

Reply:
Thanks, added this in a new commit.